pytorch_lightning/trainer/data_loading.py

```diff
@@ -89,13 +104,18 @@ def init_val_dataloader(self, model):
         :return:
         """
         self.get_val_dataloaders = model.val_dataloader
+        self.num_val_batches = 0
 
         # determine number of validation batches
         # val datasets could be none, 1 or 2+
         if self.get_val_dataloaders() is not None:
+            if not 0. <= self.val_percent_check <= 1.:
```

Comment:
as it is a relative pattern you may move this check to a private method and call it here instead so when the text is changed later it will be stil consistent...

Reply:
Done